citar.el

```diff
@@ -912,7 +912,9 @@ into the corresponding reference key.  Return
   (when (and citar-library-paths
              (stringp citar-library-paths))
     (message "Make sure 'citar-library-paths' is a list of paths"))
-  (let* ((key-entry-alist (citar--ensure-entries keys-entries))
+  (let* ((embark-default-action-overrides
+          '((consult-multi . citar-open-multi)))
```

Comment:
I think @localauthor has been right that we also need the file and url overrides then in case you act on single candidates. Otherwise you will still get the injection problem. Did you test acting on single candidates with RET and acting on all candidates with the default action?

Reply:
> Did you test acting on single candidates with RET and acting on all candidates with the default action?

Yes; worked fine.

Should I add them anyway?

Reply:
Hmm. I am kind of confused right now. I have to think about it.

Reply:
Did you load `embark-consult` in your tests?

Reply:
> Did you load `embark-consult` in your tests?

I had forgot to check that, but just now added `(require 'embark-consult)`; still no errors.

Should I require `embark-consult` then?

Reply:
Note I do get the embark error on `citar-open-library-files`, I just realized. That's because I haven't added the override there.

Reply:
> Should I require embark-consult then?

No, we will merge the multi-category renaming soon.

Reply:
So pleased to return from holiday to find all of this merged! :)

A few small things:

It seems the resources are not being propertized correctly in `citar-select-resources`, leading to all resources being categorized as "general." This means the embark-file and embark-url maps are not available when `embark-act`-ing on individual resources. 

Also, `citar-file-open` needs to `expand-file-name`, since they are abbreviated in `citar-select-resources`.

I'm working up a small PR with these changes. I'll also include the fresh `multi-category` type as well.

Reply:
Make sure you branch or rebase off latest main, as I've resolved some of these since I merged this.